src/i18n/keymap_extras/index.js

```diff
@@ -0,0 +1,60 @@
+import us from './keymap_us';
+import uk from './keymap_uk';
+import german from './keymap_german';
+import russian from './keymap_russian';
+
+/* ==Template==
+ * keymap_<lang>: {
+ *
+ *  prefix: (string) 2-letter code,
+ *
+ *  sendstring: (string) The name of the associated sendstring header file if it exists.
+ *              This is important for the "host_language" field of keymap.json in case text macros are used.
+ *
+ *  isANSI: (boolean) True if the layout is ANSI, false if not.
+ *
+ *  locales: (array[string]) List of locales, as defined in RFC 5646, closely tied to this OS keyboard layout.
+ *           This can be useful to guess the OS keyboard layout in use based on the user's preferred locale.
+ *
+ *  keycodeLUT: (HashMap<string, HashMap<string, string>) A hashmap mapping US keycode aliases like "KC_A" or "KC_DLR" or even
+ *              "QK_GESC" to a keycode object containing properties such as "name" (aka the key legends to display) and
+ *              "title" (aka the extra information shown when hovering over the key).
+ *  }
+ */
+
+export default {
+  keymap_us: {
+    prefix: 'KC',
+    // Special scenario where sendstring is an empty string not because there is no associated
+    // sendstring header file but because this is the default.
+    sendstring: '',
+    isANSI: true,
+    locales: ['en-US', 'en'],
+    keycodeLUT: us
+  },
+  keymap_uk: {
+    prefix: 'UK',
+    sendstring: 'uk',
+    isANSI: false,
+    locales: ['en-UK', 'en-GB', 'en'],
```

Comment:
I'm not sure if 'en-UK' is even a possible locale value for web navigator.

Reply:
@zvecr any idea?

Reply:
Only ever encountered `en-gb`, though take that with a pinch of salt.

Reply:
Any usage of `en-UK` would be wrong; the officially-designated IETF language tag for British English is `en-GB`.

Reply:
Ok, removed in ac3f3c216ee486d8ef7193fe29